src/compressor/Compressor.h

```diff
@@ -73,11 +73,47 @@ class Compressor {
   QatAccel qat_accel;
 #endif
 
-  static const char* get_comp_alg_name(int a);
-  static boost::optional<CompressionAlgorithm> get_comp_alg_type(const std::string &s);
+  static inline const char* get_comp_alg_name(int a) {
```

Comment:
(I know it was in the original code. Still:)
can we change 'a' into something more meaningful?

Reply:
`a` deserves a better name.

Reply:
and why did you move this method to the header file? i cannot find any explanation in the commit message.

Reply:
@tchaikov 
`crimson-alienstore` uses compressor without crimson flags whereas `compression_onwire` compiles with crimson flags. The previous signature was without `inline` and caused collisions in crimson build 
Can you recommend another way of solving this collison?